source/dealii/dealii_matrix_free_hierarchy_helpers.cc

```diff
@@ -12,6 +12,7 @@
 #include <mfmg/common/instantiation.hpp>
 #include <mfmg/common/operator.hpp>
 #include <mfmg/dealii/amge_host.hpp>
+#include <mfmg/dealii/amge_host.templates.hpp>
```

Comment:
Comment it is included for `MatrixFreeAgglomerateOperator` and that it would probably not be a bad idea to move the definition elsewhere.